webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
```

Comment:
It's better to have separate files for dev, prod, and common config.
So that this approach will be more flexible and clean when you extend config with more rules and plugins.
For more details look here:
https://github.com/survivejs/webpack-merge